CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- `POST /alert/designatedevice` endpoint (CU-hjwazd).
+- `GET /alert/location` endpoint (CU-hjwazd).
+- A helper function to format Express Validation errors.
```

Comment:
@mariocimet , could you please take another look at this pull request? I added a helper function for Express Validation error output that I plan to use in Buttons and Sensors wherever we're using Express Validation because I was finding it hard to read the error messages while testing these two new API endpoints.